app/controllers/concerns/pubmed_search.rb

```diff
@@ -2,8 +2,8 @@ module PubmedSearch
   extend ActiveSupport::Concern
 
   def search_pubmed(search_term)
-    @search_terms = search_term.split.join("+")
-    search_url = 'http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&retmode=json&retmax=20&term='+@search_terms
+    @search_term = search_term.split.join("+")
+    search_url = 'http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&retmode=json&retmax=20&term='+@search_term
```

Comment:
Also, I of course forgot that this is already a concern. But it might be better off as a service object in its own class. To be honest, I'm not sure how to use `ActiveSupport::Concern`